schema/backwards_compatibility_test.go

```diff
@@ -100,8 +100,7 @@ func TestBackwardsCompatibilityImageIndex(t *testing.T) {
          "digest": "sha256:fb2fc0707b86dafa9959fe3d29e66af8787aee4d9a23581714be65db4265ad8a",
          "platform": {
             "architecture": "arm64",
-            "os": "linux",
-            "variant": "armv8"
```

Comment:
another idea is to use Linux `utsname` (`armv8l`, `armv7l`) across all the platforms, although Linux-specific https://github.com/opencontainers/image-spec/pull/632#discussion_r109226040